src/nft/components/bag/BagRow.tsx

```diff
@@ -0,0 +1,291 @@
+        justifyContent="space-between"
+        color={grayscale ? 'darkGray' : 'blackBlue'}
+      >
+        <Column>
+          <Row overflow="hidden" whiteSpace="nowrap" gap="2">
+            <Box className={styles.assetName}>{asset.name || asset.tokenId}</Box>
+          </Row>
+          <Row overflow="hidden" whiteSpace="nowrap" gap="2">
+            <Box className={styles.collectionName}>{asset.collectionName}</Box>
+            {asset.collectionIsVerified && <VerifiedIcon className={styles.icon} />}
+          </Row>
+        </Column>
+        <Row className={styles.bagRowPrice}>
+          {`${formatWeiToDecimal(
+            asset.updatedPriceInfo ? asset.updatedPriceInfo.ETHPrice : asset.priceInfo.ETHPrice
+          )} ETH`}
+        </Row>
+      </Column>
+    </Row>
+  )
+}
+
+interface PriceChangeBagRowProps {
+  asset: UpdatedGenieAsset
+  markAssetAsReviewed: (asset: UpdatedGenieAsset, toKeep: boolean) => void
+  top?: boolean
+  isMobile: boolean
+}
+
+export const PriceChangeBagRow = ({ asset, markAssetAsReviewed, top, isMobile }: PriceChangeBagRowProps) => {
+  return (
+    <Column className={styles.priceChangeColumn} borderTopColor={top ? 'medGray' : 'transparent'}>
+      <Row className={styles.priceChangeRow}>
+        {BigNumber.from(asset.updatedPriceInfo?.ETHPrice).gt(BigNumber.from(asset.priceInfo.ETHPrice)) ? (
+          <SquareArrowUpIcon />
+        ) : (
+          <SquareArrowDownIcon />
+        )}
+        <Box>{`Price ${
+          BigNumber.from(asset.updatedPriceInfo?.ETHPrice).gt(BigNumber.from(asset.priceInfo.ETHPrice))
+            ? 'increased'
+            : 'decreased'
+        } from ${formatWeiToDecimal(asset.priceInfo.ETHPrice)} ETH`}</Box>
+      </Row>
+      <BagRow asset={asset} removeAsset={() => undefined} isMobile={isMobile} />
+      <Row gap="12" justifyContent="space-between">
+        <Box
+          className={styles.removeButton}
+          onClick={(e: MouseEvent) => {
+            e.preventDefault()
+            e.stopPropagation()
+            markAssetAsReviewed(asset, false)
+          }}
+        >
+          Remove
+        </Box>
+        <Box
+          className={styles.keepButton}
+          onClick={(e: MouseEvent) => {
+            e.preventDefault()
+            e.stopPropagation()
+            markAssetAsReviewed(asset, true)
+          }}
+        >
+          Keep
+        </Box>
+      </Row>
+    </Column>
+  )
+}
+
+interface UnavailableAssetsHeaderRowProps {
+  assets?: UpdatedGenieAsset[]
+  clearUnavailableAssets: () => void
+  didOpenUnavailableAssets: boolean
+  setDidOpenUnavailableAssets: (didOpen: boolean) => void
+  isMobile: boolean
+}
+
+interface UnavailableAssetsPreviewProps {
+  assets: UpdatedGenieAsset[]
+}
+
+const UnavailableAssetsPreview = ({ assets }: UnavailableAssetsPreviewProps) => (
+  <Column
+    display="grid"
+    style={{
+      gridTemplateColumns: `repeat(${assets.length}, 20px)`,
+      width: `${32 + (assets.length - 1) * 20}px`,
```

Comment:
we should set variable names for 20 and 32 to signify they represent row width and spacing